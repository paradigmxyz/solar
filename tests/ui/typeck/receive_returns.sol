contract F {
    receive() external payable returns (uint256) {}
    //~^ ERROR: expected one of `;`, `external`, `internal`, `override`, `payable`, `private`, `public`, `pure`, `view`, `virtual`, or `{`, found keyword `returns`
}
