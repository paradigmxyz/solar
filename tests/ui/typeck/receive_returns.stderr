error: expected one of `;`, `external`, `internal`, `override`, `payable`, `private`, `public`, `pure`, `view`, `virtual`, or `{`, found keyword `returns`
  --> ROOT/tests/ui/typeck/receive_returns.sol:LL:CC
   |
LL |     receive() external payable returns (uint256) {}
   |                                ^^^^^^^ expected one of 11 possible tokens
   |

error: aborting due to 1 previous error

